shared/test/test_table_limits.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'test_helper'
+require 'fake_redis'
+require 'redis'
```

Comment:
this should be `require 'fakeredis'`, without the `require 'redis'`, if the goal is to use the fakeredis gem